src/index.ts

```diff
@@ -316,6 +314,9 @@ export const staticPlugin = async <Prefix extends string = '/prefix'>(
                             statCache.set(path, status)
                         }
 
+                        if (!indexHTML && status.isDirectory())
```

Comment:
Return early if we don't indexHTML and the request is for a directory. This is for the test that failed after fixing the incorrect prefix.